thingsboard_gateway/connectors/can/can_connector.py

```diff
@@ -439,15 +439,20 @@ def __parse_config(self, config):
                                     self.get_name(), tb_key, config_key, )
                         continue
 
-                    if msg_config.get("command", "") and node_id not in self.__commands:
+                    if msg_config.get("command", ""):
                         cmd_config = self.__parse_command_config(msg_config["command"])
                         if cmd_config is None:
                             log.warning("[%s] Ignore '%s' %s configuration: wrong command configuration",
                                         self.get_name(), tb_key, config_key, )
                             continue
 
                         cmd_id = cmd_config["value"]
-                        self.__commands[node_id] = cmd_config
+
+                        if node_id not in self.__commands:
+                            self.__commands[node_id] = cmd_config
+                        else:
+                            # TODO: warn user if previous cmd_config differs from current cmd_config
+                            pass
```

Comment:
Complete this code block

Reply:
Added check and log message with warning level. Also added unit test for both scenarious.